app/components/leagueStandingTable/leagueStandingTable.tsx

```diff
@@ -0,0 +1,26 @@
+import { Table } from "../baseComponents";
+
+import styles from "./leagueStandingTable.module.scss";
+
+export default async function LeagueStandingTable({ contestantsScores }:{contestantsScores: any}){
+    const tableColumnNames: any[] = ["Rank", "Name", "Score"];
+    const tableData = {
+        columnNames: tableColumnNames,
+        rows: contestantsScores.at(-1).contestantRoundData
```

Comment:
fft: I get that `contestantScores` is kind of a weird catch all data object at the moment... but this blind indexing is a small to me. I wonder if we just get this passed in... and/or add a comment wherever this operation actually takes place. My preference would probably be both, but at least a comment might be nice 😁